### ISSUE TITLE
Question on Classification Subnet

### ISSUE THREAD

**Issue by DingGit** [fizyr/keras-retinanet#90]
Hi, thank you for the great work! 
I have a question on the classification subnet in this implementation.
In the original paper, it says "Finally sigmoid activations are attached to output the KA binary predictions per spatial location". However, in "default_classification_model", the output is reshaped to (-1, A) before sigmoid is applied. Shouldn't it be reshaped to (-1, KA) according to the paper? Or maybe I'm missing something here, any help is greatly appreciated!

**Comment by yhenon** [comment]
>In the original paper, it says "Finally sigmoid activations are attached to output the KA binary predictions per spatial location"

This is what we do. You can see that in line 61-62:
```
    outputs = keras.layers.Conv2D(
        filters=num_classes * num_anchors,
```
The reshaping op has no effect on how the `sigmoid` activation operates, since it operate on all elements independently.

>However, in "default_classification_model", the output is reshaped to (-1, A) before sigmoid 

The output is in fact reshaped to `(-1, num_classes)`. For the implementation, we concatenate a long list of all the anchors for simplicity.

I'm going to close this since I believe there is currently no issue here. Note that you can use the #keras-retinanet slack on the keras slack for further questions: https://keras-slack-autojoin.herokuapp.com/

**Comment by DingGit** [comment]
Yes, you are correct, thank you!
